mission-control/docs/playbooks/actions/exec.md

```diff
@@ -1,24 +1,34 @@
 # Exec Action ![Exec Icon](https://raw.githubusercontent.com/flanksource/flanksource-ui/main/src/icons/console.svg)
```

Comment:
@adityathebe Can we add examples for each action? Also use `<Icon name="console"/>`